### DISCUSSION TITLE
Non fabric support

### DISCUSSION THREAD

**Discussion by walterholohan** [software-mansion/react-native-screens#1660] [General]
I just noticed that in order to upgrade to 3.18 and above you need to have RN 0.70, does this mean you also need to enable fabric or you can still use non-fabric whilst on 0.70. Just wondering as if we need to enable fabric that would mean we need to much sure all our native libs also have migrated to fabric

**Comment by kkafar** [comment]
If you are not using Fabric the library should work with React Native versions back to 0.64.x. 

Table from [this section of readme](https://github.com/software-mansion/react-native-screens#support-for-fabric) informs that if you want to use Fabric and `react-native-screesn` at the same time -> then you must use React Native of given version or newer. 

E.g. If you want to use `react-native-screens@3.18.2` with Fabric, then you must use `react-native@0.70.0` or newer. 

> Just wondering as if we need to enable fabric that would mean we need to much sure all our native libs also have migrated to fabric

Yes. In order to use new architecture all your dependencies (that contain native components) must have Fabric support (be migrated to new architecture).
